helm/ray/values.yaml

```diff
@@ -106,7 +106,7 @@ head:
 worker:
   # If you want to disable the default workergroup
   # uncomment the line below
-  disabled: true
+  disabled: false
```

Comment:
I'd set this to `true` by default. It becomes quite a bit more complicated to set this correctly I'd say, and in a lot of cases we only run the `head` pod.

Reply:
so you are saying that the syntho ui can just work with only head? I was thinking that workers are needed. If it wasn't the case, then I'll change it to false.

Basically, our aim should be to deploy the entire stack and expect that everything just works, even though it is potentially slow, but just works. Is that the case with only `head`?